docs/deploy-development-rp.md

```diff
@@ -137,6 +137,17 @@
      journalctl _COMM=aro -o json --since "15 min ago" -f | jq -r 'select (.COMPONENT != null and (.COMPONENT | contains("access"))|not) | .MESSAGE'
      ```
 
+## Create clusters using local ARO-RP + Hive
+If you want to create clusters using the local ARO-RP + Hive instead of using the standard cluster creation process, you can do so by setting the environment variables defined in pkg/util/liveconfig/manager.go:
+- Set *ARO_ADOPT_BY_HIVE* and *ARO_INSTALL_VIA_HIVE* to *true*.
```

Comment:
Why do we need to set ADOPT variable to true when we are creating cluster via hive?

Reply:
I have updated the PR to better reflect in the docs that *ARO_ADOPT_BY_HIVE* is not needed for the cluster creation (with a more curated explanation). Tks!